### ISSUE TITLE
Issue comments aren't being rewritten

### ISSUE THREAD

**Issue by sammcj** [sammcj/rewritable-pr-comment#31]
### 🐞 Describe the bug

These test messages should be written rather than creating new messages each time.

https://github.com/sammcj/rewritable-pr-comment/issues/2#issuecomment-1356295331

**Comment by sammcj** [comment]
It looks like the user/repo values are flowing through correctly.

Looking at 

I can see the URL check was:

```
https://api.github.com/repos///pulls/comments/test-output-comment-rewritable-action
```

I would have expected that to be `/repos/sammcj/rewritable-pr-comment/pulls/`


---

Full URI

```
https://api.github.com/repos///pulls/comments/test-output-comment-rewritable-action?payload=%5Bobject%20Object%5D&eventName=push&sha=2d879ac1b309f2c827fe913714ce88f0e1be3aae&ref=refs%2Fheads%2Fmain&workflow=Build%2C%20Test%2C%20Bump%20Version%20and%20Release&action=__self&actor=sammcj&job=build-test-publish&runNumber=90&runId=3753038440&apiUrl=https%3A%2F%2Fapi.github.com&serverUrl=https%3A%2F%2Fgithub.com&graphqlUrl=https%3A%2F%2Fapi.github.com%2Fgraphql&pull_number=2
```

**Comment by sammcj** [comment]
fixed and v1.0.0 is released!

https://github.com/sammcj/rewritable-pr-comment/compare/v0.5.8...v1.0.0
